flang/lib/Lower/OpenMP/Clauses.cpp

```diff
@@ -524,7 +523,23 @@ Bind make(const parser::OmpClause::Bind &inp,
   return Bind{/*Binding=*/convert(inp.v.v)};
 }
 
-// CancellationConstructType: empty
+CancellationConstructType
+make(const parser::OmpClause::CancellationConstructType &inp,
+     semantics::SemanticsContext &semaCtx) {
+  auto name = std::get<parser::OmpDirectiveName>(inp.v.t);
+  CLAUSET_ENUM_CONVERT( //
```

Comment:
Stray `//` here? Also the `clang-format off` and `clang-format on` comments are intentional?

Reply:
This is just to make it look nicer. The same is done for the `make` function immediately above this one. Clang format doesn't understand the MS macro. There are no separators between each invocation of the macro and so clang format will try to indent each successive use with 4 more spaces than the previous one as it thinks they form part of the same expression.

I coppied the stray `//` from the example above this one. That is normally used to prevent clang-format from moving merging the current line with the following line, but it seems it does not do that in this case so I will remove the `//`.